Examples/restcomm-olympus/app/src/main/java/org/restcomm/android/olympus/Util/Utils.java

```diff
@@ -103,7 +104,7 @@ public final static boolean isValidEmail(CharSequence target) {
         params.put(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_ENABLE_PUSH_FOR_ACCOUNT, prefs.getBoolean(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_ENABLE_PUSH_FOR_ACCOUNT , true));
         params.put(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_PUSH_DOMAIN, prefs.getString(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_PUSH_DOMAIN ,""));
         params.put(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_HTTP_DOMAIN, prefs.getString(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_HTTP_DOMAIN ,""));
-        params.put(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_FCM_SERVER_KEY, prefs.getString(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_FCM_SERVER_KEY ,""));
```

Comment:
I'd say let's continue retrieving the value from preferences, but update SigninActivity with BuildConfig. TEST_PUSH_FCM_KEY to make it cleaner and more consistent. Otherwise we run the risk of trying to update prefs in signin activity and having no effect

Reply:
Sure